demo-app/src/app/modal/modal.component.scss

```diff
@@ -1 +1,5 @@
 @import "../../_route-animation-host";
+
+h5, h6 {
```

Comment:
why on h5 and h6 only?
why not on h1, h2 and h3 ?

Reply:
Materialize already make the padding top to zero for h1, h2, h3 and h4.